### PR TITLE
Add GATECH.

### PR DESCRIPTION
**Pull request by Phrohdoh**
[wikia](http://cnc.wikia.com/wiki/Battle_Lab_%28Red_Alert_2_Allied%29)
[art](https://github.com/Phrohdoh/oramod-ra2/blob/master/vanilla-rules/art.ini#L802)
[rules](https://github.com/Phrohdoh/oramod-ra2/blob/master/vanilla-rules/rules.ini#L8387)

### CODE REVIEW

File: rules/structures.yaml [hunk 1]
```diff
@@ -317,6 +317,33 @@ GADEPT:
...
+GATECH:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 60
+		Prerequisites: gaairc, gaweap, ~structures.allies
+	Selectable:
+		Bounds: 96, 98, -2, -24
+	Valued:
+		Cost: 500
```
Comment (abcdefg30) [inline rules/structures.yaml]: 2000?

Comment (Phrohdoh) [inline rules/structures.yaml]: Oops. I mixed up health and cost values.

File: rules/structures.yaml [hunk 2]
```diff
@@ -317,6 +317,33 @@ GADEPT:
 	Power:
 		Amount: -25
 
+GATECH:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 60
```
Comment (abcdefg30) [inline rules/structures.yaml]: 70, please.

**Comment by abcdefg30** [comment]
:+1: except for the bounds.

**Pull request merged**